packages/components/src/SmartObject.tsx

```diff
@@ -48,112 +48,6 @@ const SmartObjectValues = ({ smartObject }: any) => {
   )
 }
 
-const revToId = (rev: string) => rev?.split(":")[0]
```

Comment:
We should add the functions revToId and MetaData back in but comment them out for now